libraries/ArduinoOTA/examples/OTAConfigErase/WifiHealth.h

```diff
@@ -0,0 +1,49 @@
+#ifndef WIFI_HEALTH_H
+#define WIFI_HEALTH_H
+
+bool printLocalTime(Print& oPrint, time_t timeIn = 0, const char *fmt = "%I:%M:%S %p %Z, %a %b %d, %Y");
+bool printLocalTime(Print& oPrint, time_t timeIn, const char *fmt);
+extern bool WifiUp;
+
+typedef struct _WIFI_HEALTH {
+  int32_t connected_count;
+  int32_t rssi_sum;
+  uint32_t rssi_count;
+  time_t gtime_adjust;
+  time_t connected_time;
+  time_t uptime_sum;
+  time_t uptime_max;
+  time_t uptime_min;
+  time_t disconnected_time;
+  time_t downtime_sum;
+  time_t downtime_max;
+  time_t downtime_min;
+  uint8  bssid[6];
+  uint8_t channel;
+  int8_t rssi_max;
+  int8_t rssi_min;
+  int8_t rssi;
+} wifi_health_t;
```

Comment:
Why aren't these members initialized by default?
Some of them need to be initialized at values other than 0 (e.g. the "min" values and the "rssi_max"), or else it is quite easy to make mistakes.

Reply:
Thanks for reviewing!
C++ is still new to me. My experience lies with C. I didn't realize you could now initialize inside a structure under C++. I know classes and structures have strong similarities. I have never seen a structure initializing elements directly before. I searched and found some examples. It certainly makes complex initialization like this easier. Thanks for the pointer.